Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -454,6 +454,46 @@ class CredentialsManagerSpec: QuickSpec {
                         }
                     }
                 }
+                
+                it("should yield new credentials when credentials expire in the future") {
+                    credentials = Credentials(accessToken: AccessToken, tokenType: TokenType, idToken: IdToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: 3600))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        expect(credentialsManager.hasExpired(credentials)).to(beFalse())
+                        credentialsManager.renew { error, newCredentials in
+                            expect(newCredentials?.accessToken) == NewAccessToken
+                            expect(newCredentials?.refreshToken) == RefreshToken
+                            expect(newCredentials?.idToken) == NewIdToken
+                            
+                            credentialsManager.credentials { error, storedCredentials in
+                                expect(storedCredentials?.accessToken) == newCredentials?.accessToken
+                                expect(storedCredentials?.refreshToken) == newCredentials?.refreshToken
+                                expect(storedCredentials?.idToken) == newCredentials?.idToken
+                                done()
+                            }
+                        }
+                    }
+                }
+                
+                it("should yield new credentials when credentials have expired") {
```

Comment:
Similar to first, it doesn't matter if it's expired or not as the credential manager is not doing this automagically, you are forcing the refresh.